example_configs/kafka-connect.yml

```diff
@@ -0,0 +1,106 @@
+lowercaseOutputName: true
+rules:
+  #kafka.connect:type=app-info,client-id="{clientid}"
+  #kafka.consumer:type=app-info,client-id="{clientid}"
+  #kafka.producer:type=app-info,client-id="{clientid}"
+  - pattern: 'kafka.(.+)<type=app-info, client-id=(.+)><>start-time-ms'
```

Comment:
You can remove this, it's redundant with process_start_time_seconds.

Reply:
But when I load a new connector on the service Kafka Connect, already running, and it will have a different value. e.g:

This is value when  the Kafka Connect was started:
```
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.585181117777E9
(...)
# HELP kafka_connect_start_time Kafka connect JMX metric start time ms
# TYPE kafka_connect_start_time gauge
kafka_connect_start_time{clientId="connect-1",} 1.585181123694E12
```
The process_start_time_seconds is the same of kafka_connect_start_time with clientId "connect-1". But when I load a new connector the value is different for the new connector:
```
# HELP kafka_producer_start_time Kafka producer JMX metric start time ms
# TYPE kafka_producer_start_time gauge
kafka_producer_start_time{clientId="connector-producer-CsvSpoolDir-1",} 1.585183750783E12
kafka_producer_start_time{clientId="connector-producer-CsvSpoolDir-0",} 1.585183750783E12
kafka_producer_start_time{clientId="producer-3",} 1.585183744324E12
kafka_producer_start_time{clientId="producer-1",} 1.585183743829E12
kafka_producer_start_time{clientId="producer-2",} 1.585183744154E12
```

Reply:
Ah, in that case use factor to convert it to seconds and add seconds as the unit.